src/fps/example/runnerConfig.toml

```diff
@@ -0,0 +1,31 @@
+[status]
+draft = 0
+ready = 1
+contingency = 2
+signed = 3
+executed = 4
+cancelled = 5
+
+[[tasks]]
+name = "task-00"
+status = 1
```

Comment:
This feels like an additional flag that we have to manage (change every time we execute etc). Is it possible to figure out the tasks state by looking at the README of a given task? 

What we basically want here is "all non-terminal" tasks. I implemented something to this effect against the old setup, see [here](https://github.com/ethereum-optimism/superchain-ops/pull/420/files#diff-fb15a51c0ce99dc85c2344094d827b878a45fa5140a175a9f7e10ca3a8a1ea3dR17). If we could automatically determine the tasks to run tests against that would be a much better UX.